### PR TITLE
WIP: Indicate push-to-mute in tray

### LINKED ISSUE #2736: Push-to-Mute hotkey should indicate when it is active
I tried to use the "Push-to-Mute" function implemented in mumble and assigned it the hotkey 'K':
mumble-pushmute

As you see I use 'F2' to toggle muting and it works just fine.
As a second function I want to be muted _while_ I'm pressing 'K', and this doesn't appear to work at all :/

I use voice activation btw.

### PR DESCRIPTION
**Pull request by Piratonym**
This changes the tray icon when the push-to-mute hotkey is pressed. Fixes #2736.

This still needs icons, as I am not experienced with graphic design and was not sure how they should look. I would be grateful if someone could provide some.

### CODE REVIEW

File: src/mumble/MainWindow.cpp [hunk 1]
```diff
@@ -66,6 +66,7 @@ MainWindow::MainWindow(QWidget *p) : QMainWindow(p) {
 	qiIconMuteSelf.addFile(QLatin1String("skin:muted_self.svg"));
 	qiIconMuteServer.addFile(QLatin1String("skin:muted_server.svg"));
 	qiIconMuteSuppressed.addFile(QLatin1String("skin:muted_suppressed.svg"));
+	qiIconMutePushToMute.addFile(QLatin1String("skin:muted_pushtomute.svg"));
```
Comment (mkrautz) [inline src/mumble/MainWindow.cpp]: For this to work, you would need to add the icon to `mumble.qrc`: https://github.com/mumble-voip/mumble/blob/master/src/mumble/mumble.qrc

Comment (Piratonym) [inline src/mumble/MainWindow.cpp]: Interesting, it worked for me without that when testing. I'll add it.

**Comment by mkrautz** [comment]
How about these icons?

SVGs in...
[pushtomute.zip](https://github.com/mumble-voip/mumble/files/759251/pushtomute.zip)

Classic:
muted_pushtomute2


Theme:
muted_pushtomute_theme

**Comment by Nuc1eoN** [comment]
@mkrautz LGTM. So yellow would be the indicator for client side, 

@Kissaki What do you think?

**Comment by Kissaki** [comment]
Yeah, icons look good to me.

I just noticed the (dark) theme uses orange checkmark for the authenticated icon.This should probably be revisited if/once we establish orange for local status.

**Comment by Kissaki** [comment]
Will build and test before landing

File: themes/MumbleTheme.qrc [hunk 2]
```diff
@@ -39,6 +39,7 @@
     <file alias="muted_self.svg">Mumble/muted_self.svg</file>
     <file alias="muted_server.svg">Mumble/muted_server.svg</file>
     <file alias="muted_suppressed.svg">Mumble/muted_suppressed.svg</file>
+    <file alias="muted_pushtomute.svg">Mumble/muted_suppressed.svg</file>
```
Comment (Kissaki) [inline themes/MumbleTheme.qrc]: File does not exist in the theme yet.

**Comment by Nuc1eoN** [comment]
I see this has been implemented in recent snapshots, however instead of the yellow icon:
mute
I get a green one. Is this intended?

**Comment by Kissaki** [comment]
Can reproduce. Does not display new icon on push to mute.

**Comment by Kissaki** [comment]
Do you use the Mumble Lite or Dark theme?

**Comment by Nuc1eoN** [comment]
Lite theme

**Comment by Kissaki** [comment]
Those two use their own icons (see https://github.com/mumble-voip/mumble-theme). If you select "none" you should see the default mouth icon.

**Pull request merged**